services/importShop1.js

```diff
@@ -0,0 +1,109 @@
+const axios = require('axios')
```

Comment:
Correct me if I'm wrong, but it seems the crawlers are dedicated to certain websites. Maybe you could make it clear by renaming the files or including comments

Reply:
What naming do you mean? Should I name the shop?

Reply:
Since crawling is controversial, it is probably not the best idea, true. But abbreviations like acronyms or initialisms could help here.

Reply:
basically you are right. but for the reasons mentioned above, i prefer this naming convention